### ISSUE TITLE
Cannot open audio: AttributeError: 'NoneType' object has no attribute 'start'

### ISSUE THREAD

**Issue by AnWildDuck** [Zulko/moviepy#393]
### Code:
    import moviepy
    audio = moviepy.editor.AudioFileClip('audiofil.mp3')

### Error
```
Traceback (most recent call last):
  File "C:\Users\Johnny\AppData\Local\Programs\Python\Python35\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 300, in ffmpeg_parse_infos
    tbr = float(line[match.start():match.end()].split(' ')[1])
ValueError: could not convert string to float: '90k'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Johnny\Desktop\mov\speed_up.py", line 6, in <module>
    audio = moviepy.editor.AudioFileClip('audio\\Scheming Weasel faster.mp3')
  File "C:\Users\Johnny\AppData\Local\Programs\Python\Python35\lib\site-packages\moviepy\audio\io\AudioFileClip.py", line 63, in __init__
    buffersize=buffersize)
  File "C:\Users\Johnny\AppData\Local\Programs\Python\Python35\lib\site-packages\moviepy\audio\io\readers.py", line 56, in __init__
    infos = ffmpeg_parse_infos(filename)
  File "C:\Users\Johnny\AppData\Local\Programs\Python\Python35\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 305, in ffmpeg_parse_infos
    result['video_fps'] = float(line[match.start():match.end()].split(' ')[1])
AttributeError: 'NoneType' object has no attribute 'start'
```

**Comment by Gloin1313** [comment]
Can you post the audio file it fails on please? Is it only one, or have you tried several? Have you tried updating moviepy to the latest version?

**Comment by earney** [comment]
issue #448 has the same issue, and they have supplied an audio file so we can reproduce the error.
In the zip file that request we download look at the file 01\ Demacia\ Rising.mp3.

```
_clip = AudioClipFile(r"01 Demacia Rising.mp3")
```

The code above will reproduce the error.

**Comment by earney** [comment]
fixed with pull request #449.   Try pulling from repo, and if issue still exists, let us know.
